.github/stale.yaml

```diff
@@ -0,0 +1,21 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 60
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 10
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - pinned
+  - security
+  - discuss
```

Comment:
We don't have these three labels. Do we need them? Perhaps the `discuss` one, otherwise I would remove these from here.

Reply:
will remove